.github/workflows/release.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 10.x
+          node-version: 14
 
       - name: Download Cached Deps
```

Comment:
Let's remove the cache step here, we don't want to pollute the cache for jobs that run on a lower node version.